compiler/native/expand.go

```diff
@@ -78,32 +79,44 @@ func (c *client) ExpandSteps(s *yaml.Build, tmpls map[string]*yaml.Template) (ya
 			return yaml.StepSlice{}, yaml.SecretSlice{}, yaml.ServiceSlice{}, err
 		}
 
-		// skip processing template if the type isn't github
-		if tmpl.Type != "github" {
-			logrus.Errorf("Unsupported template type: %v", tmpl.Type)
-			continue
-		}
-
-		// parse source from template
-		src, err := c.Github.Parse(tmpl.Source)
-		if err != nil {
-			return yaml.StepSlice{}, yaml.SecretSlice{}, yaml.ServiceSlice{}, fmt.Errorf("invalid template source provided for %s: %v", step.Template.Name, err)
-		}
+		// nolint:ineffassign // ignoring linter on overwriting the "bytes" var
```

Comment:
it's claiming this directive is not needed here.. can we remove?

Reply:
Yeah, I'm kinda confused on this on... VSCode keeps telling me I need it and the linter above complained. idk.. I'll just remove it and see what happens again I guess. 😢